bedrock/products/templates/products/vpn/platforms/desktop.html

```diff
@@ -41,124 +41,113 @@
-      class='vpn-content-well-right-half'
+      class_name='vpn-content-media-right-half',
     ) %}
       <p>{{ ftl('vpn-desktop-wifi-copy') }}</p>
     {% endcall %}
 
-    {% call vpn_platform_content_well(
-      title=ftl('vpn-desktop-privacy-headline'),
+    {% call vpn_content_media(
+      heading=ftl('vpn-desktop-privacy-headline'),
       image_url='img/products/vpn/platforms/vpn-cntwell-one-click.png',
-      class='vpn-content-well-left-half'
+      class_name='vpn-content-media-left-half'
     ) %}
       <p>{{ ftl('vpn-desktop-privacy-copy') }}</p>
     {% endcall %}
 
-    {% call vpn_platform_content_well(
-      title=ftl('vpn-desktop-servers-headline'),
+    {% call vpn_content_media(
+      heading=ftl('vpn-desktop-servers-headline'),
       image_url='img/products/vpn/platforms/vpn-cntwell-globe.png',
-      class='vpn-content-well-right-half'
+      class_name='vpn-content-media-right-half',
     ) %}
       <p>{{ ftl('vpn-desktop-servers-copy', servers=vpn_servers) }}</p>
     {% endcall %}
 
-    {% call vpn_platform_content_well(
-      title=ftl('vpn-desktop-devices-headline', devices=vpn_devices),
+    {% call vpn_content_media(
+      heading=ftl('vpn-desktop-devices-headline', devices=vpn_devices),
       image_url='img/products/vpn/platforms/vpn-cntwell-devices.png',
-      class='vpn-content-well-left-half'
+      class_name='vpn-content-media-left-half'
     ) %}
       <p>{{ ftl('vpn-desktop-devices-copy') }}</p>
     {% endcall %}
 
-    {% call vpn_platform_content_well(
-      title=ftl('vpn-desktop-speed-headline'),
+    {% call vpn_content_media(
+      heading=ftl('vpn-desktop-speed-headline'),
       image_url='img/products/vpn/platforms/vpn-cntwell-speed-guage.png',
-      class='vpn-content-well-right-half'
+      class_name='vpn-content-media-right-half',
     ) %}
       <p>{{ ftl('vpn-desktop-speed-copy') }}</p>
     {% endcall %}
 
-    {% call vpn_platform_content_well(
-      title=ftl('vpn-desktop-log-headline'),
+    {% call vpn_content_media(
+      heading=ftl('vpn-desktop-log-headline'),
       image_url='img/products/vpn/platforms/vpn-cntwell-privacy.png',
-      class='vpn-content-well-left-half'
+      class_name='vpn-content-media-left-half'
     ) %}
       <p>{{ ftl('vpn-desktop-log-copy') }}</p>
     {% endcall %}
 
-    <!-- Custom content well for the CTA -->
-    <section class="vpn-content-well-cta">
-      <div class="vpn-content-well-cta-container">
-        <h2 class="vpn-content-well-title">{{ ftl('vpn-platforms-shared-cta-headline') }}</h2>
-        <div class="vpn-content-well-desc">
-          <a class="vpn-button" href="{{ url('products.vpn.landing') }}">{{ ftl('vpn-platforms-shared-cta') }}</a>
-          <div class="vpn-hero-guarantee">
-            <em>{{ ftl('vpn-platforms-shared-day-guarantee') }}</em>
-          </div>
-        </div>
+    {% call vpn_content_block(
+      class_name='t-highlight platform-get-started-cta'
```

Comment:
Using the `vpn_content_block` call as seen in the `pricing-fixed.html` include. I added a modifer class to center the elements (per JR's direction).

Reply:
Is there a more generic name we could use for this modifier class, something that's not associated with the pages where it is used?

Reply:
I think so – it's basically just `t-center`.

Reply:
Renamed to `.t-center`